tox.ini

```diff
@@ -148,7 +148,7 @@ setenv=
   lvm_osds: CEPH_STABLE_RELEASE = luminous
 deps=
   ansible2.2: ansible==2.2.3
-  ansible2.3: ansible==2.3.1
+  ansible2.3: ansible==2.4.0
```

Comment:
"ansible2.3" now points at "2.4.0". would you please explain why we don't add a new "ansible2.4" line here instead?

My guess is that it's so that Jenkins will run the "ansible2.3" scenarios quickly? What will the final commit to master look like?